src/main/scripts/org/reaktivity/specification/nukleus/oauth/control/route/proxy/resolve.one.realm.with.set.roles.issuer.and.audience.multiple.routes.then.route.proxy/controller.rpt

```diff
@@ -0,0 +1,96 @@
+#
+# Copyright 2016-2019 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property nukleiR05C ${nuklei:directory("target/nukleus-itests").controlCapacity(4096, 4096)}
+
+property controlR05C ${nukleiR05C.control()}
+
+property nukleiRG010101C ${nuklei:directory("target/nukleus-itests").controlCapacity(4096, 4096)}
+
+property controlRG010101C ${nukleiRG010101C.control()}
+
+connect "agrona://stream/bidirectional"
+        option agrona:reader ${agrona:broadcastReceiver(controlR05C.controller)}
+        option agrona:writer ${agrona:manyToOneWriter(controlR05C.nukleus)}
+
+connected
+
+write 0x00000011
+write ${controlR05C.nextCorrelationId()}
+write ${core:string("oauth")}
+write [0x05] "RS256"
+write 21
+write [0x06] "scope1"
+write [0x06] "scope2"
+write [0x06] "scope3"
+write ${oauth:resolveEx()
+             .issuer("test issuer")
+             .audience("testAudience")
+             .build()}
+write flush
+
+read 0x40000011
+read ${controlR05C.correlationId()}
+read [0..8] # authorization
+
+write 0x00000001
+write ${controlRG010101C.nextCorrelationId()}
+write ${core:string("oauth")}
+write [0x02]
+write 0x0000_000000000000L
+write ${core:string("oauth#0")}
+write ${core:string("target#0")}
+write flush
+
+read 0x40000001
+read ${controlRG010101C.correlationId()}
+read (long:newProxyRouteId)
+
+write 0x00000011
+write ${controlR05C.nextCorrelationId()}
+write ${core:string("oauth")}
+write [0x05] "RS256"
+write 21
+write [0x06] "scope1"
+write [0x06] "scope2"
+write [0x06] "scope3"
```

Comment:
Should this have fewer scopes to resolve to a different `authorization` for the second `ROUTE`?